recipes/vulkan-loader/all/conanfile.py

```diff
@@ -0,0 +1,151 @@
+                              "if(FALSE)")
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["VULKAN_HEADERS_INSTALL_DIR"] = self.deps_cpp_info["vulkan-headers"].rootpath
+        self._cmake.definitions["BUILD_TESTS"] = False
+        self._cmake.definitions["USE_CCACHE"] = False
+        if self.settings.os == "Linux":
+            self._cmake.definitions["BUILD_WSI_XCB_SUPPORT"] = self.options.with_wsi_xcb
+            self._cmake.definitions["BUILD_WSI_XLIB_SUPPORT"] = self.options.with_wsi_xlib
+            self._cmake.definitions["BUILD_WSI_WAYLAND_SUPPORT"] = self.options.with_wsi_wayland
+            self._cmake.definitions["BUILD_WSI_DIRECTFB_SUPPORT"] = self.options.with_wsi_directfb
+        if self.settings.os == "Windows":
+            self._cmake.definitions["ENABLE_WIN10_ONECORE"] = False
+        if tools.is_apple_os(self.settings.os):
+            self._cmake.definitions["BUILD_STATIC_LOADER"] = not self.options.shared
+        self._cmake.definitions["BUILD_LOADER"] = True
+        if self.settings.os == "Windows":
+            self._cmake.definitions["USE_MASM"] = True
+        self._cmake.configure()
+        return self._cmake
+
+    def build(self):
+        if self.deps_cpp_info["vulkan-headers"].version != self.version:
+            raise ConanInvalidConfiguration("vulkan-loader must be built with the same version than vulkan-headers.")
+        self._patch_sources()
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+        self.copy("LICENSE.txt", dst="licenses", src=self._source_subfolder)
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "loader"))
+
+    def package_info(self):
+        if self.deps_cpp_info["vulkan-headers"].version != self.version:
+            self.output.warn("vulkan-headers version is different than vulkan-loader. Several symbols might be missing.")
+
+        self.cpp_info.names["cmake_find_package"] = "Vulkan"
+        self.cpp_info.names["cmake_find_package_multi"] = "Vulkan"
+        self.cpp_info.names["pkg_config"] = "vulkan"
+        suffix = "-1" if self.settings.os == "Windows" else ""
+        self.cpp_info.libs = ["vulkan" + suffix]
+        self.cpp_info.includedirs = [] # no include dir by itself, public headers come from vulkan-headers
+        if self.settings.os == "Linux":
+            self.cpp_info.system_libs = ["dl", "pthread", "m"]
+        elif self.settings.os == "Macos":
+            self.cpp_info.frameworks = ["CoreFoundation"]
+
+        vulkan_sdk_path = self.package_folder
+        self.output.info("Create VULKAN_SDK environment variable: {}".format(vulkan_sdk_path))
+        self.env_info.VULKAN_SDK = vulkan_sdk_path
```

Comment:
I don't know if I should put this environment variable in this recipe, or in some `vulkan-sdk` interface recipe (requiring a coherent combination of versions of `vulkan-loader`, `vulkan-header`, `vulkan-validationlayers`, `shaderc`, `glslang`, `spirv-cross`, `spirv-tools` and `spirv-headers` recipes).

Reply:
Let's merge and this can be a new issue and wait for community feedback, once we have enough we can make a decision.